userspace/libsinsp/async_cgroup.h

```diff
@@ -86,15 +86,33 @@ struct delayed_cgroup_value {
 	int64_t m_memory_limit;
 };
 
+/**
+ * \brief Behavior when container_id is missing from the cgroup path
+ *
+ * Usually the container ID is present in cgroup paths and the code that reads
+ * resource limits from cgroups checks that to avoid treating e.g. root
+ * cgroup settings as per-container values.
+ *
+ * In some cases we want to skip the check and read the limits even if
+ * the cgroup path does not contain the container ID
+ */
+enum class cgroup_check
```

Comment:
this is a little awkward since it is combining two ideas, log-level and then skipping

defer